### PR TITLE
Pull together Redux store and docs for Walkthrough One

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

File: src/pages/tutorial/task/task.js [hunk 1]
```diff
@@ -94,6 +96,48 @@ class TaskPage extends React.Component {
     setProgress(progress);
   };
 
+  areDocLinksReady = () => {
```
Comment (aidenkeating) [inline src/pages/tutorial/task/task.js]: @dlabaj I've added this in as a temporary fix for the Asciidoc not re-rendering when I update the attributes. I'll look into why the Asciidoc isn't re-rendering.

File: src/services/walkthroughServices.js [hunk 2]
```diff
@@ -25,24 +27,26 @@ import {
...
     watchers: [
       {
         resource: namespace => routeDef(namespace),
-        handlerFn: handleWalkthroughOneRoutes
+        handlerFn: handleWalkthroughOneRoutes.bind(null, 'walkthrough-one')
       }
     ]
   },
   oneA: {
+    id: WALKTHROUGH_IDS.ONEA,
```
Comment (witmicko) [inline src/services/walkthroughServices.js]: nit: `ONE_A` ?

**Comment by aidenkeating** [comment]
7577c90 is an attempt to make the asciidoc component reactive. Instead of storing the compiled html we just store whatever is returned from the adoc or template endpoint and compile it on the render call. Doesn't seem to have any performance issues and it means I can remove the hacky "loading" state I had in before.

**Comment by aidenkeating** [comment]
@dlabaj Mind taking a look?

**Comment by aidenkeating** [comment]
@dlabaj Think this is ready to merge

**Pull request merged**